src/PowerShellEditorServices/Session/ExecutionTarget.cs

```diff
@@ -0,0 +1,23 @@
+namespace Microsoft.PowerShell.EditorServices.Session
```

Comment:
I think this new file needs a licensing header (at least that is needed in the main PowerShell repo). Applies to other changes as well